planet/cli/orders.py

```diff
@@ -98,44 +98,96 @@ async def cancel(ctx, order_id):
     click.echo('Cancelled')
 
 
+def split_list_arg(ctx, param, value):
+    if value is None:
+        return None
+    elif value == '':
+        # note, this is specifically checking for an empty string
+        click.BadParameter('Entry cannot be an empty string.')
+
+    # split list by ',' and remove whitespace
+    entries = [i.strip() for i in value.split(',')]
+
+    # validate passed entries
+    for e in entries:
+        if not e:
+            raise click.BadParameter('Entry cannot be an empty string.')
+    return entries
+
+
+@orders.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@click.argument('order_id', type=click.UUID)
+@click.option('--delay', type=int, default=5,
+              help='Time (in seconds) between polls.')
```

Comment:
I wonder if we might want to remove this option and start with a fixed delay? It might turn out that we want to use an exponential back-off down the road for orders that take a rather long time.

Reply:
Hmm.. this is a good thought. I think if we get to where we want to add exponential delay, we can expand the option type so that the value can either be an integer or 'exponential'. I believe this option is valuable because users may have different wait sensitivities.